app/javascript/packs/document-capture.jsx

```diff
@@ -30,6 +30,20 @@ function getServiceProvider() {
   return { name, failureToProofURL, isLivenessRequired };
 }
 
+/**
+ * @return {Record<'front'|'back'|'selfie', string>}
+ */
+function getBackgroundUploadURLs() {
```

Comment:
Would having this return an array make it more amenable to `Promise.all` kind of stuff?

```
[
  { key: 'front', url: 'url' },
  { key: 'back', url: 'url2 },
]
```

Reply:
> Would having this return an array make it more amenable to `Promise.all` kind of stuff?

Possibly, depending which direction we go with `Promise.all` based on above comments.

But it would also make it more cumbersome where we [do a lookup of these URLs in the higher-order component](https://github.com/18F/identity-idp/blob/ad50f6f2af36c1e1fa3561139182831d7faa4f0a/app/javascript/packages/document-capture/higher-order/with-background-encrypted-upload.jsx#L14) for the mirrored async upload.

To me, there's a nice overlap between the form of the form data payload object and the configured background upload URLs allowing for background upload on the intersection of keys:

```js
const backgroundUploadURLs = { front: '...', back: '...' };
const nextFormValues       = { front: '...', back: '...', foo: 'bar' };
```

Reply:
I'm inclined to keep this as-is since I've not changed anything with `resolveObjectValues`, though it could be that I've misunderstood the comment at https://github.com/18F/identity-idp/pull/4314#discussion_r506704015. Happy to reconsider.